FindTown/FindTown/Presentation/MapScene/AddressSheet/AddressSheetViewController.swift

```diff
@@ -0,0 +1,170 @@
+    
+    var viewModel: AddressSheetViewModel?
+    
+    private let titleLabel = FindTownLabel(
+                                text: "다른 동네 보기",
+                                font: .subtitle4,
+                                textColor: .black)
+    
+    fileprivate let backButton = ImageButton(imageText: "navi.back", tintColor: .grey7)
+    fileprivate let countyCollectionView = CityCollectionView()
+    fileprivate let villageCollectionView = CityCollectionView()
+    fileprivate let completeButton = FTButton(style: .largeFilled)
+    
+    private let screenWidth = UIScreen.main.bounds.width
+    
+    init(viewModel: AddressSheetViewModel) {
+        super.init(bottomHeight: screenWidth * 1.3)
+        self.viewModel = viewModel
+     }
+    
+    override init(bottomHeight: CGFloat) {
+        super.init(bottomHeight: bottomHeight)
+    }
+    
+    required init?(coder: NSCoder) {
+        super.init(coder: coder)
+    }
+    
+    override func setupView() {
+        super.setupView()
+        
+        completeButton.setTitle("선택 완료", for: .normal)
+        self.rx.cityType.onNext(.county)
+    }
+    
+    override func bindViewModel() {
+        super.bindViewModel()
+        
+        completeButton.rx.tap
+            .throttle(.milliseconds(300), scheduler: MainScheduler.instance)
+            .bind { [weak self] _ in
+                if self?.completeButton.isSelected == false {
+                    self?.viewModel?.input.didTapCompleteButton.onNext(())
+                }
+            }
+            .disposed(by: disposeBag)
+        
+        backButton.rx.tap
+            .throttle(.milliseconds(300), scheduler: MainScheduler.instance)
+            .bind { [weak self] _ in
+                self?.rx.cityType.onNext(.county)
+            }
+            .disposed(by: disposeBag)
+        
+        viewModel?.output.countyDataSource
+            .bind(to: countyCollectionView.rx.items(cellIdentifier: CityCollectionViewCell.reuseIdentifier,
+                                              cellType: CityCollectionViewCell.self)) { index, item, cell in
+                cell.setupCell(itemText: item.rawValue)
+            }.disposed(by: disposeBag)
+        
+        viewModel?.output.villageDataSource
+            .bind(to: villageCollectionView.rx.items(cellIdentifier: CityCollectionViewCell.reuseIdentifier,
+                                              cellType: CityCollectionViewCell.self)) { index, item, cell in
+                cell.setupCell(itemText: item.village)
+            }.disposed(by: disposeBag)
+        
+        countyCollectionView.rx.modelSelected(County.self)
+            .map { county in
+                return county.villages.map { City(county: county, village: $0) }
+            }
+            .subscribe(onNext: { cities in
+                self.viewModel?.output.villageDataSource.onNext(cities)
+                self.rx.cityType.onNext(.village)
+            })
+            .disposed(by: disposeBag)
```

Comment:
```suggestion
        countyCollectionView.rx.modelSelected(County.self)
            .map { county in
                return county.villages.map { City(county: county, village: $0) }
            }
            .delay(.milliseconds(400), scheduler: MainScheduler.instance)
            .subscribe(onNext: { cities in
                self.viewModel?.output.villageDataSource.onNext(cities)
                self.rx.cityType.onNext(.village)
            })
            .disposed(by: disposeBag)
```
사용자가 선택한 구를 보여주고 동 선택으로 넘어가는게 조금 더 자연스럽지 않을까 싶어서 delay를 넣는건 어떨지 제안드려봅니당..!

Reply:
좋은 것 같습니다! 바로 반영하겠습니다 👍